src/ore_algebra/ore_algebra.py

```diff
@@ -46,6 +46,7 @@
 import sys
 
 from sage.categories.algebras import Algebras
+from sage.categories.finite_field import FiniteFields
```

Comment:
Shouldn't this be `finite_fields`? Have you run the test suite?

Reply:
oops right, I will make another PR to fix that